eos-extra-resize

```diff
@@ -43,10 +50,13 @@ if [ "$marker" = "GUID:55" ]; then
     extra_part_name=${extra_part#/dev/}
     part_start=$(< /sys/class/block/$extra_part_name/start)
 
-    # The partition table is recreated below, which also includes the
-    # magic 55 attribute being dropped.
+    # The partition table is recreated below
     echo "Try to resize $extra_part at sector $part_start to fill $extra_disk"
-    echo "$part_start" | sfdisk --force --no-reread --label gpt $extra_disk
+    echo "$part_start" | sfdisk --force --no-reread --label $pt_label $extra_disk
+
+    # For MBR the marker is removed in endless-repartition.sh when rewriting
```

Comment:
Actually for the extra disk, for both GPT and MBR the marker is removed in this script 2 lines above, we rewrite the whole partition table in that sfdisk command.